epinorm/norm.py

```diff
@@ -1,40 +1,138 @@
 import json
 import pandas as pd
+import numpy as np
 import re
-
+import itertools
 from csv import QUOTE_NONE
-from epinorm.config import REF_DATA_DIR
+from transliterate import translit
+from transliterate.exceptions import LanguageDetectionError
+
 from epinorm.geo import NominatimGeocoder
-from epinorm.utils import coalesce
+from epinorm.config import MIN_ADMIN_EXCEPTIONS, REF_DATA_DIR, COUNTRIES_DATA, COUNTRIES_EXCEPTIONS, ADMIN_LEVELS_DATA
+
 
 HOST_SPECIES_FILE = REF_DATA_DIR / "ncbi_host_species.csv"
 PATHOGEN_SPECIES_FILE = REF_DATA_DIR / "ncbi_pathogen_species.csv"
 
 SAMPLING_MODES = ("top", "bottom", "random")
 OUTPUT_FILE_SEPARATOR = "\t"
 OUTPUT_COLUMNS = [
-    "observation_date",
-    "report_date",
-    "pathogen_species_ncbi_id",
-    "pathogen_species_name",
-    "pathogen_serotype",
-    "host_species_ncbi_id",
-    "host_species_name",
-    "host_species_common_name",
-    "host_domestication_status",
-    "latitude",
-    "longitude",
+    # "observation_date",
+    # "report_date",
+
+    # "pathogen_species_ncbi_id",
+    # "pathogen_species_name",
+    # "pathogen_serotype",
+
+    # "host_species_ncbi_id",
+    # "host_species_name",
+    # "host_species_common_name",
+    # "host_domestication_status",
+
+    # "latitude",
+    # "longitude",
     "country",
-    "admin_level_1",
-    "locality",
     "country_osm_id",
-    "admin_level_1_osm_id",
+    "locality",
     "locality_osm_id",
-    "original_record_source",
-    "original_record_id",
+    "admin_level_1",
+    "admin_level_1_osm_id",
+
+    # "original_record_source",
+    # "original_record_id",
     "original_record_location_description",
 ]
 
+def get_translitaration_endonym(row):
```

Comment:
Correct the typo in the function name and update all calls to the function.

```suggestion
def get_transliteration_endonym(row):
```